src/eslint-config-adeira/runner/__tests__/shouldLintAll.test.js

```diff
@@ -13,6 +13,8 @@ test.each([
   'package.json',
   'src/packages/relay/package.json',
   '/Users/code/universe/src/packages/relay/package.json',
+  '.eslintignore',
+  '/Users/code/universe/src/packages/relay/.eslintignore',
```

Comment:
Eslint ignore file which is not in the root doesn't work if I recall correctly.

Reply:
Ahh, yes, you are right